templates/rhpam75-authoring-ha.yaml

```diff
@@ -219,12 +219,17 @@ parameters:
 - displayName: DataGrid Image
   description: DataGrid image.
   name: DATAGRID_IMAGE
-  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.1
+  value: registry.redhat.io/jboss-datagrid-7/datagrid73-openshift:1.2
+  required: true
+- displayName: DataGrid Container CPU Limit
+  description: DataGrid Container cpu limit.
+  name: DATAGRID_CPU_LIMIT
+  value: "4"
```

Comment:
I think this value is not correct, CPU limits on OCP are set in millicores.
I don't know hoe much cpu does jdg requires, but, I think something around 500m would be enough.
https://docs.openshift.com/container-platform/3.11/dev_guide/compute_resources.html#dev-compute-resources

Reply:
@spolti Good catch, but hmmm... we must've had it wrong in there before. It was previously set to "0.5", which also isn't "millicores". @jakubschwan or @sutaakar : What would you suggest here, then?

Reply:
@errantepiphany @spolti this works as well, see https://docs.openshift.com/container-platform/3.11/dev_guide/compute_resources.html#dev-viewing-limit-ranges (e.g. Core Limit Range Object Definition). If you are prefer to use 4000m instead of 4 I'm ok with it.

Reply:
is there a reason to use 4cores? isn't that too much?

Reply:
@jakubschwan , @spolti How about we just set 4 as the maximum, and don't set the minimum?

Reply:
Disregard previous comment (I just deleted). Was thinking of something else.

Reply:
going with 1000m (millicores) for now.

Reply:
sounds good.